docs/guide/index.md

```diff
@@ -7,3 +7,5 @@ Welcome to the Textual Guide! An in-depth reference on how to build apps with Te
 Most of the code in this guide is fully working&mdash;you could cut and paste it if you wanted to.
 
 Although it is probably easier to check out the [Textual repository](https://github.com/Textualize/textual) and navigate to the `docs/examples/guide` directory and run the examples from there.
+
+The tree of directories under `docs/examples/` contains of hundreds of examples for all parts of Textual.  There are also complete applications in Textual's `examples/` directory. 
```

Comment:
Same comment, no need to mention docs/examples here. But examples/ is probably a good idea.